Voice/Answering Machine Detection with NodeJS/env.example

```diff
@@ -0,0 +1,14 @@
+# Project ID copied from the API Credentials in your SignalWire Space
+PROJECT_ID=
+# API token copied from the API Credentials in your SignalWire Space
+API_TOKEN=
+# Your SignalWire Space URL copied from the API Credentials in your SignalWire Space
+SIGNALWIRE_SPACE_URL=
+# The URL you would like to receive the AMD result
+SERVER_URL=
+# A phone number from your SignalWire account for caller ID in E164 format
+SW_NUMBER=
+# The test phone number to dial in E.164 format
+CUSTOMER_NUMBER=
+# A live agent's phone number
+AGENT_NUMBER=
```

Comment:
could we use the same format we use in the style guide? like:

 - adding examples
 - SPACE_URL instead of SIGNALWIRE_SPACE_URL
 - spacing

Reply:
I have literally never noticed that we have a style guide in this examples repo